File: 268-MissingNumber/268-MissingNumber.go
// Last updated: 8/24/2026, 9:42:23 PM
func missingNumber(nums []int) int {
    s := len(nums) 
    sum := 0
    sum2 := 0

    for i:=0; i<s; i++{
        sum += nums[i] 
    }

    for i:=0; i<=s; i++{
        sum2 += i
    }

    return sum2 - sum
}

File: 268-MissingNumber/268-MissingNumber.java
// Last updated: 8/24/2026, 9:42:15 PM
class Solution {
    public int missingNumber(int[] nums) {
        int sum = 0;
        int max = nums.length;

        for (int i=0; i<nums.length; i++){
            sum += nums[i];
            max += i;
        }

    return max-sum;
    }
}